client/ProductSizeModal.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+function ProductSizeModal(props) {
+  let sizes, image, isHidden;
```

Comment:
Here I think you could declare the variables as const instead of let and assign them at the same time they're declared.

Reply:
Changes made.